travis/check-doctum-parse-errors.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# This file is part of the phpBB Forum Software package.
+#
+# @copyright (c) phpBB Limited <https://www.phpbb.com>
+# @license GNU General Public License, version 2 (GPL-2.0)
+#
+# For full copyright and license information, please see
+# the docs/CREDITS.txt file.
+#
+set -e
+
+DB=$1
+TRAVIS_PHP_VERSION=$2
+NOTESTS=$3
+
+if [ "$NOTESTS" == '1' ]
+then
+	if [ ! -f doctum.phar ]; then
+		# Download the latest release
+		curl -O https://doctum.long-term.support/releases/5.1/doctum.phar
```

Comment:
We have `doctum.phar` in the package now, shouldn\t we use it instead of downloading the latest?

Reply:
We only download if the file does not exist ;)
This an update script if you remove the file

Reply:
Well, it always should exist now :) But well.